sublime-package.json

```diff
@@ -4,18 +4,21 @@
       {
         "file_patterns": ["/LSP-json.sublime-settings"],
         "schema": {
+          "$id": "sublime://settings/LSP-json",
           "definitions": {
             "userSchema": {
               "type": "object",
               "properties": {
                 "fileMatch": {
                   "type": "array",
+                  "markdownDescription": "An array of file patterns to match against when resolving JSON files to schemas. `*` can be used as a wildcard. Exclusion patterns can also be defined and start with '!'. A file matches when there is at least one matching pattern and the last matching pattern is not an exclusion pattern.",
```

Comment:
Split this to multiple lines, so that the popup is not truncated in ST3.

Reply:
Do you have a screenshot of this behavior?

Reply:
I've understood that it's the issue with height not being adjusted properly when lines need to be wrapped. So not so much truncation but just height too small (but scrollable).

Reply:
https://github.com/sublimelsp/LSP/issues/646

Reply:
Is it this issue? https://github.com/sublimehq/sublime_text/issues/2854

Reply:
If you have two lines or more and one or more are long enough then it is truncated.

Reply:
@rwols, yeah i think it's that one.

Reply:
That's a sublime issue. Working around ST defects is ultimately bad in the long run because core issues don't get fixes. We should instead rely on SublimeHQ fixing their mistakes.

Reply:
Well it's not that much of an effort, just whenever you encounter a very long descriptions try to split it.

Reply:
They don't look nice the bug aside.